### ISSUE TITLE
Embed Google Fonts

### ISSUE THREAD

**Issue by dlabordus** [com-pas/compas-open-scd#42]
As a developer
I want to included the Google Fonts in the source
So that we don't have a dependency with Google at runtime

Background:
We currently retrieve the Icons from https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@300&family=Roboto:wght@300;400;500&display=swap
In some environments that isn't allowed, because it's a external source.

Some info to check is
https://google-webfonts-helper.herokuapp.com/fonts/roboto-mono?subsets=latin

**Comment by Sander3003** [comment]
@ca-d is this some thing to summit upstream to OpenSCD?

**Comment by ca-d** [comment]
> @ca-d is this some thing to summit upstream to OpenSCD?

Absolutely, thanks for asking! We're looking forward to your pull request, don't hesitate to ask if you need any help.
